src/views/AppRoot/__tests__/component.spec.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import { shallow } from 'enzyme'
+
+import AppRootComponent from '../component'
+
+describe('AppRootComponent matches snapshot', () => {
+  const defaultProps = {
+    isLogged: false
+  }
+
+  it('with default props', () => {
+    // eslint-disable-next-line react/jsx-props-no-spreading
+    const component = shallow(<AppRootComponent {...defaultProps} />)
+
+    expect(component).toMatchSnapshot()
+  })
+
+  it('with default props', () => {
+    const component = shallow(<AppRootComponent isLogged />)
```

Comment:
it would be better to name it `isLoggedIn` to avoid confusion with process known as `logging`

Reply:
Good point